geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/main/scala/org/locationtech/geomesa/fs/storage/common/utils/PartitionSchemeArgResolver.scala

```diff
@@ -6,14 +6,14 @@
  * http://www.opensource.org/licenses/apache2.0.php.
  ***********************************************************************/
 
-package org.locationtech.geomesa.fs.tools.utils
+package org.locationtech.geomesa.fs.storage.common.utils
```

Comment:
Ok, out of curiosity, why did this file move?

Reply:
mainly so i didn't need to pull in another dependency in nifi, but it seemed better to have it more generally available instead of tied to the tools module.